symfony/asset-mapper/6.4/importmap.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Returns the import map for this application.
+ *
+ * - "path" is a path inside the asset mapper system. Use the
+ *     "debug:asset-map" command to see the full list of paths.
+ *
+ * The "importmap:require" command can be used to add new entries to this file.
+ *
+ * This file has been auto-generated by the importmap commands.
+ */
```

Comment:
You can grab this exact language from core: https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/AssetMapper/ImportMap/ImportMapConfigReader.php#L110

Reply:
fixed, including a fix in that file as well.